src/SimpleGraphs/generators/staticgraphs.jl

```diff
@@ -763,3 +763,27 @@ function lollipop_graph(n1::T, n2::T) where {T <: Integer}
     add_edge!(g, n1, n1+1)
     return g
 end
+
+
+"""
+    friendship_graph(n)
+Create a [friendship graph](https://en.wikipedia.org/wiki/Friendship_graph)
+consisting of n copies of the cycle graph C3 with a common vertex`,
+the center vertex will always be 1, if n was <= 0 the function will return
+single node graph
+
```

Comment:
```suggestion
A friendship graphs consisting of n copies of the cycle graph C3 with a common center vertex.
This  center vertex will always be 1. If n <= 0 return a graph with a single node.
```
In general, for these docstrings its a good form, to have a single sentence on the first line describing what your function does, and then on the next line a more detailed explanation.